racetime/views/race.py

```diff
@@ -431,6 +431,11 @@ def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['category'] = self.get_category()
         kwargs['can_moderate'] = kwargs['category'].can_moderate(self.user)
+        for field in models.Race._meta.get_fields():
```

Comment:
You should iterate over the form fields rather than the model. The model isn't appropriate here as it contains fields which aren't user-settable and don't appear in the form. I think `for field in self.get_form_class()._meta.fields` will work (this gives a list of strings so you'll need to change `field.name` to just `field` too)